core/templates/dev/head/pages/exploration_editor/simple_editor_tab/simple_editor_sidebar_directive.html

```diff
@@ -112,6 +112,9 @@
   simple-editor-sidebar .highlighted {
     background-color: #c5e1a5;
   }
+  simple-editor-sidebar .error {
```

Comment:
@seanlip Yes i have searched the code whenever required but i just didnt use grepping , my editor has directory and file searches which i have been using from start to figure where a function call has been made and how different files are connected.